src/tests/ports/catalog-queries.spec.ts

```diff
@@ -13,439 +13,445 @@ import {
+            filters.maxPrice,
+          ])
         })
       })
-    })
 
-    describe('and passing the "search" filter', () => {
-      let search: string
-      beforeEach(() => {
-        search = 'a search string'
-        filters = {
-          search,
-        }
-      })
-      it('should add the is search definition to the WHERE', () => {
-        expect(getCollectionsQueryWhere(filters).text).toContain(
-          `items.search_text ILIKE '%' || $1 || '%'`
-        )
-        expect(getCollectionsQueryWhere(filters).values).toStrictEqual([
-          filters.search,
-        ])
+      describe('and passing the "onlyListing" filter', () => {
+        let onlyListing: boolean
+        beforeEach(() => {
+          onlyListing = true
+          filters = {
+            onlyListing,
+          }
+        })
+        it('should add the only listing definition to the WHERE', () => {
+          expect(getCollectionsQueryWhere(filters).text).toContain(
+            `(items.search_is_store_minter = false OR (items.search_is_store_minter = true AND available = 0)) AND listings_count > 0`
+          )
+        })
       })
-    })
 
-    describe('and passing the "rarities" filter', () => {
-      let rarities: Rarity[]
-      beforeEach(() => {
-        rarities = [Rarity.COMMON, Rarity.EPIC]
-        filters = {
-          rarities,
-        }
-      })
-      it('should add the rarities definition to the WHERE', () => {
-        expect(getCollectionsQueryWhere(filters).text).toContain(
-          `items.rarity = ANY($1)`
-        )
-        expect(getCollectionsQueryWhere(filters).values).toStrictEqual([
-          filters.rarities,
-        ])
+      describe('and passing the "onlyMinting" filter', () => {
+        let onlyMinting: boolean
+        beforeEach(() => {
+          onlyMinting = true
+          filters = {
+            onlyMinting,
+          }
+        })
+        it('should add the only minting definition to the WHERE', () => {
+          expect(getCollectionsQueryWhere(filters).text).toContain(
+            `items.search_is_store_minter = true AND available > 0`
+          )
+        })
       })
-    })
 
-    describe('and passing the "contractAddresses" filter', () => {
-      let contractAddresses: string[]
-      beforeEach(() => {
-        contractAddresses = ['0xaContractAddress']
-        filters = {
-          contractAddresses,
-        }
-      })
-      it('should add the contract addresses definition to the WHERE', () => {
-        expect(getCollectionsQueryWhere(filters).text).toContain(
-          `items.collection = ANY($1)`
-        )
-        expect(getCollectionsQueryWhere(filters).values).toStrictEqual([
-          filters.contractAddresses,
-        ])
+      describe('and passing the wearable related filters', () => {
+        let isWearableHead: boolean, isWearableAccessory: boolean
+        beforeEach(() => {
+          ;(isWearableAccessory = true), (isWearableHead = true)
```

Comment:
```suggestion
          isWearableAccessory = true
          isWearableHead = true
```

Reply:
these are not related to the PR itself, they were like this before, but if you want I can change it anyways